models/src/main/java/com/vimeo/networking2/Subscriptions.kt

```diff
@@ -0,0 +1,56 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * A model that holds the type of push subscriptions a user has. This class
+ * has both getters and setters since a user can update their subscriptions
+ * using a Patch.
+ */
+@JsonClass(generateAdapter = true)
+data class Subscriptions(
```

Comment:
This class seems to be a duplicate of `NotificationTypeCount`. Is it different?

Reply:
`NotificationTypeCount` has fields which are `int`. Each field represents the number of notifications you have by type. It is part of the `Notification` model. `Subscriptions` has fields which are booleans. Each fields represents whether you have the notification turned on or off. They are distinct models.